### PR TITLE
Update git terminology in conflict template from checkout to switch with appropriate switch

### PR DESCRIPTION
**Pull request by jmorrowomni**
Update git terminology in conflict template from checkout to switch with appropriate switch

#  #

## 🗣 Description ##

Update the old git `checkout` syntax with current `switch` syntax

## 💭 Motivation and context ##

Eliminate confusion and prevent issues with further updates if old syntax is deprecated

## 🧪 Testing ##

All steps were successfully tested locally

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

**Review by jsf9k — CHANGES_REQUESTED** [review]
You need to bump the patch portion of the version:  `./bump_version.sh patch`

**Comment by jmorrowomni** [comment]
> You need to bump the patch portion of the version: `./bump_version.sh patch`

I saw that on the PR I used for reference, and then completely forgot to do it - bumping now
🤜

File: src/lineage/templates/conflict_template.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ that you must resolve before merging this pull request!
     git remote add {{ lineage_id }} {{ remote_url }}
     git remote set-url --push {{ lineage_id }} no_push
     git switch {{ local_branch }}
-    git checkout -b {{ pr_branch_name }} --track origin/{{ local_branch }}
+    git switch -c {{ pr_branch_name }} --track origin/{{ local_branch }}
```
Comment (mcdonnnj) [inline src/lineage/templates/conflict_template.md]: Long options are preferred when available ^^

```suggestion
    git switch --create {{ pr_branch_name }} --track origin/{{ local_branch }}
```

Comment (jsf9k) [reply src/lineage/templates/conflict_template.md]: Good catch @mcdonnnj!

**Review by jsf9k — APPROVED** [review]

**Comment by jsf9k** [comment]
@jmorrowomni - Please don't forget to create a release for `v0.2.5` after this PR is merged.

**Review by mcdonnnj — APPROVED** [review]

**Pull request merged**